tools/xdp/s2n-quic-xdp/src/mmap.rs

```diff
@@ -19,6 +19,13 @@ pub struct Mmap {
     len: usize,
 }
 
+#[derive(Debug)]
+pub enum MmapOptions {
+    None,
```

Comment:
Is this option actually needed if the functions take an `Option` already?

Reply:
Completely correct. Fixed